images/build-e2e/entrypoint.sh

```diff
@@ -72,8 +77,6 @@ else
 fi
 
 # Copy crc-e2e binary and pull-secret
-# Review this when ux feature cleanup environment 
-rm "${FEATURES_PATH}/ux.feature"
```

Comment:
Not clear why this change is in the `CLEANUP_HOME` commit?

Reply:
@adrianriobo this need to be resolved.

Reply:
yeah, I am on it :+1:

Reply:
moved to the testing_mode commit as this was related with the ux mode